### DISCUSSION TITLE
CSS anchor positioning

### DISCUSSION THREAD

**Discussion by ilyam2014** [saadeghi/daisyui#3112] [Ideas / feature requests]
will CSS anchor positioning be used in components in daisyUI 5?

**Comment by saadeghi** [comment]
Probably only for dropdown and tooltip

**Comment by ilyam2014** [comment]
Yeah i meant these components, thanks for the answer

**Comment by RadithSandeepa** [comment]


**Reply by ilyam2014** [reply]
ai generated answer be like:

**Reply by RadithSandeepa** [reply]
English is not my first language, so to make my responses more formal and readable, I use GPT to refine my ideas. Even this reply is AI-generated. However, I always strive to provide accurate answers to the best of my ability.

Regarding your question, the answer is indeed available. If you have some knowledge of how to type and search, you can easily find the information yourself without needing to ask simple questions here.

Answer to your question is literally [here]( https://daisyui.com/blog/daisyui-5-upcoming-changes/)

**Reply by ilyam2014** [reply]
the post was published after i asked about this here first and i already got an answer from the creator so your reply is reduntant. more over it's ai generated lol
